client/src/pages/AboutUs.js

```diff
@@ -396,18 +399,21 @@ const AboutUs = () => {
           {t("companyLogoInfo")}
         </p>
         <h3>{t("lifetimeSupporters")}</h3>
-        <SupportersLogosContainer id="AU_LS">
+        <SupportersLogosContainer id={getGTM("lifetimeSupporters")}>
```

Comment:
@jessica-lau @ddveloper 
There's no need to add GTM tags for these containers they are not tracked. We need only ID for testing.
Also Press release button is not track as far as I remember. 
Please clarify this with @FedericaZn

Reply:
@galeza sorry I wasn't aware. I'll remove the GTM tags for the containers after @FedericaZn confirms and leave the container IDs in the aboutUS.js as is.

Reply:
my bad 😅